### PR TITLE
[Form] fix translating file validation error message

### PR DESCRIPTION
**Pull request by xabbuh**
| Q             | A
| ------------- | ---
| Branch?       | 3.4
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | 
| License       | MIT
| Doc PR        | 

I messed this up in #30895.

#FOSSHackathons #EUFOSSA

### CODE REVIEW

File: src/Symfony/Bundle/FrameworkBundle/Resources/config/form.xml [hunk 1]
```diff
@@ -93,6 +93,7 @@
             <deprecated>The "%service_id%" service is deprecated since Symfony 3.1 and will be removed in 4.0.</deprecated>
         </service>
         <service id="form.type.file" class="Symfony\Component\Form\Extension\Core\Type\FileType" public="true">
+            <argument type="service" id="translator" on-invalid="ignore" />
             <deprecated>The "%service_id%" service is deprecated since Symfony 3.1 and will be removed in 4.0.</deprecated>
```
Comment (xabbuh) [inline src/Symfony/Bundle/FrameworkBundle/Resources/config/form.xml]: Probably means we need to undeprecate the service, right?

Comment (HeahDude) [reply src/Symfony/Bundle/FrameworkBundle/Resources/config/form.xml]: Yes!

**Review by HeahDude — APPROVED** [review]
👍

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @xabbuh.

**Pull request merged**